sql/hive-thriftserver/src/test/scala/org/apache/spark/sql/hive/thriftserver/CliSuite.scala

```diff
@@ -234,6 +234,13 @@ class CliSuite extends SparkFunSuite with BeforeAndAfterAll with Logging {
     )
   }
 
+  test("unsupported operations") {
```

Comment:
@hvanhovell The latest changes added the test cases for the unsupported operations.